docs/framework/database/schema-design.md

```diff
@@ -26,6 +26,6 @@ When we look at CiviCRM Tables there are a few patterns that generally hold true
 - Every table has a id column which is Auto Increment and therefore unique key for that table
 - Columns which reference other tables generally speaking will be named in the format of `other table name` + `_id`. For example in `civicrm_country` there is `address_format_id` which indicates that is references `civicrm_address_format.id`
 - Many-to-many relationships use "join tables" as intermediary tables. For example, a contact can have many activity records, and an activity can have many contact records. So the table `civicrm_activity_contact` is used as the glue because it has foreign keys to both.
-- In some places CiviCRM defines schema using a construct called [PseudoConstants](/framework/database/schema-definition.md#table-field-pseudoconstant) which produces some slightly more complex logic
+- In some places CiviCRM defines schema using a construct called [PseudoConstants](/docs/framework/database/schema-definition.md#table--field--pseudoconstant-table-field-pseudoconstant) which produces some slightly more complex logic
```

Comment:
Similarly, this link *is* working for me as-is. Is it not working for you?

Note this point [here](https://docs.civicrm.org/dev/en/latest/documentation/markdown/#internal-url-standards): 

> Use absolute URLs which begin with a forward slash. (The root directory is the `docs` folder.)

Perhaps I should highlight that point more prominently? I realize that perhaps it's confusing.